### PR TITLE
Arcade general release blog post

### LINKED ISSUE #6416: Arcade Announcement Blog Post
Galen, can you post this to the blog on Monday?
Thanks!
Files located here - https://microsoft-my.sharepoint.com/:f:/p/jacqchen/ElLMNzAIOBdPsdkR_khczwwB_-4sHdRNvT-BxZKRf_TJ8A?e=6vBmYO

### PR DESCRIPTION
**Pull request by ganicke**
General release announcement blog post for Arcade.

Combined from @Jaqster 's draft and #6373.

Closes #6416.

### CODE REVIEW

File: docs/blog/arcade/general-release.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+## Coding curriculum
+
+The MakeCode Arcade curriculum is freely available at https://arcade.makecode.com/courses/csintro. This curriculum covers Computer Science and game development concepts through a series of activities and lessons. Also, there are plenty of game tutorials, lessons, and samples available: https://arcade.makecode.com/docs. 
+ 
+![Lesson to choose from](/static/blog/arcade/general-release/lessons.png)
+
+## Connecting with us
+
+If you find any issues, or have suggestions, please log them on GitHub (https://github.com/microsoft/pxt-arcade/issues). If you have questions, or would like to participate in the MakeCode community, please join the Forum (https://forum.makecode.com) or follow us on social [@MSMakeCode](https://twitter.com/MSMakeCode).
```
Comment (ganicke) [inline docs/blog/arcade/general-release.md]: Is "social" a term now or say "social media"??

...follow us on social **media** [@MSMakeCode](https://twitter.com/MSMakeCode).

File: docs/blog/arcade/general-release.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# MakeCode Arcade Released for General Availability!
+
+**Posted on January 6th, 2019 by [Jaqster](https://github.com/jaqster)**
+
+Ringing in the New Year with a New Release! The MakeCode team is very happy and proud to announce that MakeCode Arcade has been officially released for general availability today!
+
+For those of you who have been following our progress, you’ll know that we’ve been working on MakeCode Arcade for about 2 years now... and we’ve had it in Beta for the past year while we’ve continued to get feedback, fix bugs, make improvements and just generally polish up the experience.
+
+Huge thanks to everyone who has been using the Beta version, filing bugs and offering us suggestions (shout-outs to Kevin J Walters and Alex Kulcsar for helping us find a lot of great bugs!). Also big thanks to our *amazing* hardware partners who have worked with us to make super fun Arcade-compatible game devices that really bring the Arcade games to life!  Check them out here – https://arcade.makecode.com/hardware.
```
Comment (ganicke) [inline docs/blog/arcade/general-release.md]: Should we also tag Kevin and Alex's twitter's or something?

Comment (Jaqster) [reply docs/blog/arcade/general-release.md]: I don't know their handles/contact info...

**Comment by Jaqster** [comment]
Hey, can we publish the blog post today?

File: docs/blog/arcade/general-release.md (hunk 1 above)
Comment (Jaqster) [reply docs/blog/arcade/general-release.md]: "social media" is fine...

**Comment by ganicke** [comment]
Sure, merge it in.

**Pull request merged**